lib/utils/CrossfilterHistogramPlot.ts

```diff
@@ -188,6 +223,16 @@ export function CrossfilterHistogramPlot(
 	let node = svg.node();
 	assert(node, "Infallable");
 
+	node.addEventListener("mousemove", (event) => {
+		const relativeX = event.clientX - node.getBoundingClientRect().left;
+		const value = x.invert(relativeX);
+		const f = d3.format("~s");
+		hovered.value = f(value);
```

Comment:
I might be wrong, but doesn't d3.format() return a string? I'm guessing since deno isn't being helpful for you locally, this TS error isn't showing up.

I think we want:

```suggestion
		hovered.value = x.invert(relativeX);
```

and then above to format only when we render to the DOM.

Reply:
Yup, you're right! Changed it now.